src/test/java/net/datafaker/providers/base/BoardgameTest.java

```diff
@@ -0,0 +1,23 @@
+package net.datafaker.providers.base;
+
+import java.util.Collection;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Please use specific imports

Reply:
The import is even completely superfluous. Thanks for the tip!